src/components/app/TabBar.js

```diff
@@ -43,9 +49,23 @@ class TabBar extends React.PureComponent<Props> {
               })}
               key={tabSlug}
               data-name={tabSlug}
-              onMouseDown={this._mouseDownListener}
+              onClick={this._onClickListener}
             >
-              {tabsWithTitle[tabSlug]}
+              {/* adding a button for better keyboard navigation  
+            (allows tabbing between the different panels)
+            ARIA attributes for better accessibility and usability
+            (arrow navigation between the tabs) */}
```

Comment:
The second part of the comment isn't quite right: the ARIA attributes don't do anything by themselves besides exposing some data to the screenreader. Can you please remove this part then? (the part about ARIA attributes)

Also, could you please add in a comment an explanation that the id will be used in panels' `aria-labelledby` attribute. This is because we don't use IDs for CSS and therefore this can be surprising to see an ID here.